### PR TITLE
fix INetChannelInfo

### PR DESCRIPTION
**Pull request by Lidbetter**

### CODE REVIEW

**Comment by number201724** [comment]
Did the game update and add these two back?

File: public/inetchannelinfo.h [hunk 1]
```diff
@@ -67,11 +67,9 @@ class INetChannelInfo
...
 	virtual float		GetAvgLatency( void ) const = 0;	// average packet latency in seconds
 	virtual float		GetEngineLatency( void ) const = 0;	// current latency (RTT), more accurate but jittering
-	
-	virtual void		unk001( void ) const = 0;
-	virtual void		unk002( void ) const = 0;
-	
-	virtual float		GetAvgLoss( int flow ) const = 0;	 // avg packet loss[0..1]
+
+    virtual float		GetAvgLoss(int flow) const = 0;	 // avg packet loss[0..1]
+    virtual float		GetAvgLatePackets(int flow) const = 0;	 // avg late packets[0..1]
```
Comment (Lidbetter) [inline public/inetchannelinfo.h:72]: I'm not sure if this is actually late packets - its something like that

```
// set on server not client
net_fakejitter 10 100 20

                              Send     Recv    Total   See also
Latency. . . . . . . . :       0ms      0ms      0ms   net_fakelag
Loss . . . . . . . . . :     0.00%    0.00%            net_fakeloss
Reorder: Percentage  . :     0.00%    0.00%
         Delay . . . . :         -        -
Jitter:  Average . . . :   10.00ms  10.00ms            net_fakejitter
         Max . . . . . :   20.00ms  20.00ms
         Percentage. . :   100.00%  100.00%

GetAvgLossIn 0.000000
GetAvgLossOut 0.000000

GetAvgLatePacketsIn 0.347222
GetAvgLatePacketsOut 0.199000
```

**Comment by Lidbetter** [comment]
> Did the game update and add these two back?

no, when this was changed for the 2024-10-02/03 update, it was not quite right

I may still be missing another func, if I am its somewhere between `GetTotalPackets` and the end

**Comment by GAMMACASE** [comment]
Currently this pr makes from 3 virtual funcs (unk001, unk002, GetAvgLoss) down to 2 (GetAvgLoss, GetAvgLatePackets), making a vtable shift by one of any other further functions, is that expected?

**Comment by GAMMACASE** [comment]
Should be resolved now with https://github.com/alliedmodders/hl2sdk/pull/279

**Pull request closed without merging**